ocs_ci/ocs/utils.py

```diff
@@ -677,6 +680,24 @@ def setup_ceph_toolbox(force_setup=False):
             'json --patch  \'[{ "op": "replace", "path": '
             '"/spec/enableCephTools", "value": true }]\''
         )
+    # use the existing toolbox but fix the keyring to talk to
+    # external cluster
+    independent_mode = ocsci_config.DEPLOYMENT.get("independent_mode")
+    if independent_mode:
+        keyring_dict = ocsci_config.INDEPENDENT_MODE.get("admin_keyring")
+        log.info("Setting up keyring on toolbox for independent mode")
+        ceph_toolbox = get_pod_name_by_pattern('rook-ceph-tools', namespace)
+        from ocs_ci.ocs.resources.pod import upload
```

Comment:
Is the import here purposeful or can we import at the top of the file?

Reply:
Resolving because the new changes removed this